vsintegration/tests/unittests/Tests.ProjectSystem.MultiTargeting.fs

```diff
@@ -30,8 +30,8 @@ type MultiTargeting() =
         let dirName = Path.GetDirectoryName(projFile)
         let libDirName = Directory.CreateDirectory(Path.Combine(dirName, "lib")).FullName
         let codeBase = (new Uri(Assembly.GetExecutingAssembly().EscapedCodeBase)).LocalPath |> Path.GetDirectoryName
-        let refLibPath = Path.Combine(libDirName, "FSharp.Core.Unittests.dll")
-        File.Copy(Path.Combine(codeBase, "FSharp.Core.Unittests.dll"), refLibPath)
+        let refLibPath = Path.Combine(libDirName, "VisualFSharp.Unittests.dll")
+        File.Copy(Path.Combine(codeBase, "VisualFSharp.Unittests.dll"), refLibPath)
```

Comment:
fsharp.core. unit tests test fsharp.core.  VisualFSharp unit tests are the ide unit tests.

The dependence here , was merely that the tests needed a dll to reference.  So I let them use visualfsharp.unittest.dll instead.

if one ran ````build vs```` on a clean machine 3 tests would fail.